docs/src/main/sphinx/release/release-417.md

```diff
@@ -0,0 +1,40 @@
+# Release 417 (10 May 2023)
+
+## General
+
+* Improve performance of `UNION ALL` queries. ({issue}`17265`)
```

Comment:
I don't think this is accurate. Does it also benefit queries involving `UNION DISTINCT` ? @sopel39 ?